internal/services/webhook_cache.go

```diff
@@ -59,6 +60,13 @@ func (wc *WebhookCache) Update(newData map[uint32]map[string][]Webhook) {
 	wc.mu.Lock()
 	defer wc.mu.Unlock()
 	wc.webhooks = newData
+	var total int
+	for _, m := range newData {
+		for _, hooks := range m {
+			total += len(hooks)
+		}
+	}
+	fmt.Printf("Webhook cache updated: %d webhook configurations loaded\n", total)
```

Comment:
if this will be logging on the server, let's use the zerolog logger so that kibana can index it. Regular strings logged like this don't get properly picked up since kibana expects our standard JSON document format.